mxcubeweb/core/components/queue.py

```diff
@@ -1821,8 +1821,11 @@ def execute_entry_with_id(self, sid, tindex=None):
                 not len(current_queue[sid]["tasks"])
             ) and sid != self.app.lims.get_current_sample().get("sampleID", ""):
                 try:
-                    self.app.sample_changer.mount_sample(current_queue[sid], wait=True)
+                    self.app.mxcubecore.get_adapter("sample_changer").mount_sample(
+                        current_queue[sid], wait=True
+                    )
```

Comment:
I recommend having as little code as possible in the `try` block, so that we are more precise about what is acceptable to catch and ignore at runtime, as opposed to what should straight up crash MXCuBE.

```suggestion
                sample_changer = self.app.mxcubecore.get_adapter("sample_changer")
                try:
                    sample_changer.mount_sample(
                        current_queue[sid], wait=True
                    )
```

Reply:
Not sure I think thats a good idea at this stage. I think its perfectly fine to fail the queue if for some reason `get_adapter("sample_changer")` would return None. If this would happen we don't want it to be raised further. Logging it is completely fine.